README.md

```diff
@@ -60,7 +60,8 @@ The event name and what's in the `data` for each event handler:
   * `senderId` The ID of the sender
   * `session` [A Session object](#the-session-object) you can mutate
   * `source` One of `quickReply`, `postback`, `text`
-  * `text` Message content, `event.message.text` for text events, `payload` for `postback` and `quickReply` events
+  * `text` Normalized message content, `event.message.text` for text events, `payload` for `postback` and `quickReply` events
```

Comment:
Going to tweak this a little bit since `text` in `message` events is not normalized.